### DISCUSSION TITLE
Add ability to edit bodies of Discussions posts

### DISCUSSION THREAD

**Discussion by hwittenborn** [community/community#4953] [Mobile]
If I try to edit a post I made under a discussion, it currently won't allow me to, and will only give me the option to quote the reply.

I'm almost questioning if it's a bug, but it's something I'd really like implemented regardless.

**Comment by eliperkins** [comment]
Hey @hwittenborn, thanks for this feedback.

Discussion comments are editable by tapping on the `•••` button to the top-right of the specific comment. In that menu, you should see an "Edit" option.

If there's a place that you'd expect to see it where you're not seeing it, feel free to let us know and we'll look into that as a bug report!

**Comment by hwittenborn** [comment]
Hi!

Here's a screenshot of what I'm seeing on my end:

![Capture+_2021-08-10-15-31-46(1).png](https://user-images.githubusercontent.com/74838472/128931046-76e5e55b-3348-4382-9164-4123b6203658.png)

I see the edit button when using the web interface, just not when using the mobile app.

**Reply by eliperkins** [reply]
Ah, perfect! I think I'm able to reproduce this locally now. I'll let you know when the fix for this is rolling out!
